src/composables/useDocumentTitle.ts

```diff
@@ -99,10 +99,12 @@ export function useDocumentTitle() {
 			}
 
 			if ((lastMessage.actorId === actorId.value && lastMessage.actorType === actorType.value)
-				|| (isArchived && !hasUnreadMentions(conversation) && !hasCall(conversation))) {
+				|| (isArchived && !hasUnreadMentions(conversation) && !hasCall(conversation))
+				|| lastMessage.systemMessage) {
```

Comment:
This is incorrect as per chat last week. Just because the last message is a system message, doesn't mean all messages we loaded are.